src/coreclr/tools/r2rdump/CoreDisTools.cs

```diff
@@ -24,23 +24,23 @@ public enum TargetArch
             Target_Arm64
         };
 
-        [DllImport(_dll)]
+        [DllImport(_dll, CallingConvention = CallingConvention.Cdecl)]
         public static extern IntPtr InitBufferedDisasm(TargetArch Target);
 
-        [DllImport(_dll)]
-        public static extern void DumpCodeBlock(IntPtr Disasm, ulong Address, IntPtr Bytes, int Size);
+        [DllImport(_dll, CallingConvention = CallingConvention.Cdecl)]
+        public static extern void DumpCodeBlock(IntPtr Disasm, IntPtr Address, IntPtr Bytes, int Size);
 
-        [DllImport(_dll)]
+        [DllImport(_dll, CallingConvention = CallingConvention.Cdecl)]
         [return: MarshalAs(UnmanagedType.I4)]
```

Comment:
```suggestion
```
Unnecessary

Reply:
Great, thanks for confirming, I was also wondering what's the use of this clause ;-).

Reply:
Removed in 2nd commit.